src/components/Nav.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import Link from "next/link";
+import Button from "./Button";
+import { useRouter } from "next/navigation";
+import { NAV_LINKS } from "@/constants";
+import { useEffect } from "react";
+import { useStore } from "@/app/store";
+const toastr = require("toastr");
```

Comment:
Not a big fan of using a `require` here. We should definitely try to disable this and have linting errors around this if we can.

Also, lets use a toaster from https://ui.shadcn.com/docs/components/toast